### DISCUSSION TITLE
App performance question

### DISCUSSION THREAD

**Discussion by jtermi03** [sct/overseerr#961] [Help] [answered]
So I've gotten everything up and running and have successfully gone end to end, very exciting! One question: as this is my first time hosting something like this, I'm not really sure where to look first, but it takes about 10-15 seconds for search results to load most of the time. Is that a question of how many resources I've allocated my container/VM or just something that improves over time as the application matures?

**Comment by sct** [comment, marked as answer]
It shouldn't take 10-15 seconds, let alone more than 1 second, to do anything really. It sounds like something is misconfigured? Network issue, perhaps? I don't believe this is anything wrong with Overseerr.

**Reply by jtermi03** [reply]
hm, [here is a video](https://drive.google.com/file/d/1av12luItQ8gQSdnyqrFOTtKhCUom5pki/view?usp=sharing) of the network tab open when loading Overseerr and below is my setup, maybe something will stand out?

- Overseerr running in a Docker container in an Ubuntu VM on TrueNAS 12. 
- Nginx Proxy Manager in the same container
- VM has 2048MB RAM and 1 core, using around 700 MB. 
- Plex, Sonarr, and Radarr in separate jails on the same TrueNAS server
- A library of 1300 movies and 1000 tv episodes
- 1000/1000 UL/DL network speed (wired directly to router)
- A domain bought on namecheap.com and DNS services through Cloudflare

Tested on a desktop in the local network and on two mobile devices with WiFi off with similar results

I followed these instructions to the t: https://smarthomepursuits.com/setup-overseerr/

**Reply by TheCatLady** [reply]
@jtermi03 That definitely isn't normal, but it is likely the result of some issue with your setup (like @sct said, possibly a network issue?), and not Overseerr itself.

We pull media metadata and images from TMDb, so you might want to check if there are any problems accessing their API endpoints from your network.

Unfortunately we do not support third-party guides/configurations, but you're welcome to pop into our [Discord server](https://discord.gg/PkCWJSeCk7) and ask if anyone else is experiencing similar issues.
